api/groups/networkGroup.go

```diff
@@ -314,6 +327,32 @@ func (ng *networkGroup) getRatingsConfig(c *gin.Context) {
 	)
 }
 
+// getGenesisNodesConfig return genesis nodes configuration
+func (ng *networkGroup) getGenesisNodesConfig(c *gin.Context) {
+	start := time.Now()
+	eligibleNodesConfig, waitingNodesConfig := ng.getFacade().GetGenesisNodesPubKeys()
+	log.Debug(fmt.Sprintf("GetGenesisNodesConfig took %s", time.Since(start)))
+
+	if eligibleNodesConfig == nil || waitingNodesConfig == nil {
```

Comment:
could have returned an error from the ng.getFacade().GetGenesisNodesPubKeys() as to signal the operation failed.
This is slightly cleaner as we do not need to always remember to test both returned values. Just an error, the if it is not nil it just failed and that's it.

Reply:
done